cli/src/main/java/org/jboss/as/cli/CommandLineMain.java

```diff
@@ -21,27 +21,34 @@
 */
 package org.jboss.as.cli;
 
+import java.io.File;
+import java.io.FileWriter;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.Locale;
 import java.util.Properties;
 import java.util.logging.LogManager;
 
 import org.jboss.as.cli.impl.CliLauncher;
+import org.jboss.logging.Logger;
 import org.jboss.logmanager.Configurator;
 import org.jboss.logmanager.LogContext;
 import org.jboss.logmanager.PropertyConfigurator;
 
-
 /**
 *
 * @author Alexey Loubyansky
 */
 public class CommandLineMain {
+    private static final Logger LOG = Logger.getLogger(CommandLineMain.class);
```

Comment:
We're going to have to re-think this part and not use a static logger. This will be created before the `configureLogMangaer()` is invoked and could create issues.

Reply:
https://github.com/wildfly/wildfly-core/pull/5550